### DISCUSSION TITLE
run vanilla commands on signs and make it cost money

### DISCUSSION THREAD

**Discussion by ludaseu** [EssentialsX/Essentials#4549] [Q&A] [answered]
is there a way to make a sign run a command and take money from the person who used it
for example if I make a sign and when someone interacts with the sign it runs /locate village and tells them where the nearest village is then it takes $500 from their balance. is there anyway to do that

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of Essentials as we do not provide ways to run commands from signs let alone charge for them.

**Reply by ludaseu** [reply]
ah thank you for telling me
